### PR TITLE
Susi css branch

### PR DESCRIPTION
**Pull request by susiirwin**
Updates Navigation, with User name, and link to user dashboard. adds some styling to the order tables as well as a hover on the order table. Navigation is now done using a partial in the shared folder

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -9,26 +9,8 @@
...
-      <li role="presentation">
-        <% if !cart_empty? %>
-          <%= link_to "Cart", cart_path %>
-        <% end %>
-      </li>
-    </ul>
-  </div>
-</div>
+
+<%= render 'shared/navigation' %>
```
Comment (PlanetEfficacy) [inline app/views/layouts/application.html.erb]: Great idea moving this out.

File: app/views/shared/_navigation.html.erb [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+    <ul class="nav nav-tabs">
+      <a class="navbar-brand" href="/">Waddams Emporium</a>
+      <li role="presentation">
+        <% if logged_in? %>
+          <%= link_to "Welcome, #{current_user.username}...", dashboard_path, class: "nav-item nav-link" %>
+          <%= link_to "Checkout", orders_path, class: "nav-item nav-link" %>
+          <%= link_to "Logout", logout_path, method: :delete, class: "nav-item nav-link"%>
+        <% else %>
+          <%= link_to "Login", login_path, class: "nav-item nav-link"%>
+          <%= link_to "Create Account", new_user_path, class: "nav-item nav-link"%>
```
Comment (PlanetEfficacy) [inline app/views/shared/_navigation.html.erb]: I like this button here.

File: app/views/shared/_navigation.html.erb [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
-    </li>
-  </ul>
+<div class="container">
+  <div class="nav">
+    <ul class="nav nav-tabs">
+      <a class="navbar-brand" href="/">Waddams Emporium</a>
+      <li role="presentation">
+        <% if logged_in? %>
+          <%= link_to "Welcome, #{current_user.username}...", dashboard_path, class: "nav-item nav-link" %>
+          <%= link_to "Checkout", orders_path, class: "nav-item nav-link" %>
```
Comment (PlanetEfficacy) [inline app/views/shared/_navigation.html.erb]: I think we want to only show checkout if a user is logged in and has items in the cart.

File: db/schema.rb [hunk 4]
```diff
@@ -57,6 +57,7 @@
   add_index "orders", ["user_id"], name: "index_orders_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
+    t.string   "name"
```
Comment (PlanetEfficacy) [inline db/schema.rb]: User already has a name with the user_profile table. Any reason it belongs here instead / in addition?

File: app/views/shared/_navigation.html.erb (hunk 3 above)
Comment (susiirwin) [inline app/views/shared/_navigation.html.erb]: I agree - Let me see about getting some logic in there :)

File: app/views/shared/_navigation.html.erb (hunk 2 above)
Comment (susiirwin) [inline app/views/shared/_navigation.html.erb]: Thanks!

File: db/schema.rb (hunk 4 above)
Comment (susiirwin) [inline db/schema.rb]: SO there should not have been an addition - when i did the merge from the latest dev branch into this branch, there was a merge conflict in this file of an empty line. i had to delete the HEAD and merge conflict notes in the schema file around the empty line to get rid of it.

**Pull request merged**